draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1201,61 +1229,89 @@ uCDN-A                         dCDN-B                         dCDN-C
    during processing and passed on by the dCDN to any further dCDNs in a
    cascade.</t>
 
-	<section title="CI/T Objects" anchor="sect-5.1"><t>
+	<section title="CI/T Objects" anchor="cit-objects"><t>
    The top-level objects defined by the CI/T interface are described in
    this section.</t>
 
 	<t>
    The encoding of values used by these objects is described in
-   <xref target="sect-5.2"/>.</t>
-
-	<section title="CI/T Trigger Command" anchor="sect-5.1.1"><t>
-   CI/T Trigger Command, which is used to create a new trigger, MUST use a MIME media type of "application/cdni; ptype=ci-trigger-command.trigger.v2".</t>
+   <xref target="cit-objects-properties"/>.</t>
 
+	<section title="CI/T Trigger Command" anchor="cit-trigger-create-command">
 	<t>
-   A CI/T Trigger Command is encoded as a JSON object containing the following
-   name/value pairs.</t>
+	CI/T Create Trigger Command, which is used to create a new trigger, 
+	MUST use a MIME media type of "application/cdni; ptype=ci-trigger-command.create.v2".
+	</t>
+	<t>
+   	A CI/T Create Trigger Command is encoded as a JSON object containing the following
+	name/value pairs.
+	</t>
 
 	<t><list style="empty" hangIndent="3">
-	<t><list style="hanging" hangIndent="3"><t hangText="Name: trigger">
+	<t><list style="hanging" hangIndent="3"><t hangText="Name: trigger-spec">
 	<vspace blankLines="1"/>
 	Description: A specification of the trigger action and a set of
          targets upon which to act.
 	<vspace blankLines="1"/>
-	Value: A Trigger.v2 Specification, as defined in <xref target="sect-5.2.1"/>.
+	Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
 	<vspace blankLines="1"/>
 	Mandatory: Yes.
 	</t>
 
 	<t hangText="Name: cdn-path">
 	<vspace blankLines="1"/>
-	Description: The CDN PIDs of CDNs that have already issued the
-         CI/T Command to their dCDNs.
+	Description: The CDN PIDs of CDNs that have already issued the CI/T Command to their dCDNs.
 	<vspace blankLines="1"/>
-	Value: A non-empty JSON array of JSON strings, where each
-         string is a CDN PID as defined in <xref target="sect-4.6"/>.
+	Value: A non-empty JSON array of JSON strings, where each string is a CDN PID as defined in 
+	<xref target="loop-detection-and-prevention"/>.
 	<vspace blankLines="1"/>
-	Mandatory: Yes.
+	Mandatory: No.
```

Comment:
Would it be perhaps safer to say that this is not mandatory, but recommended in use cases where CDN loops could occur (i.e. not Open Caching)?

Reply:
Added:
 uCDN SHOULD specify "cdn-path" in those use cases when CDN loop detection is a concern (See <xref target="loop-detection-and-prevention"/> for details).